File: 9-palindrome-number/palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        int reverse = 0;
        int xcopy = x;
        while(x>0){
            int digit = x%10;
            reverse = (reverse*10) + digit;

            x = x/10;
        }

        if(reverse == xcopy){
            return true;
        }else{
            return false;
        }
    }
}

File: 9-palindrome-number/palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        num = 0
        temp = x
        if x<0:
            return False
        else:
            while temp>0:
                digit = temp%10
                num = num*10+digit
                temp//=10
            if x == num:
                return True
            else:
                return False
                